Update xknx to 0.15.3 (#15301)
home-assistant/home-assistant.github.io: source/_integrations/knx.markdown

  required: false
  type: string
reset_after:
  description: Reset back to OFF state after specified seconds.
  required: false
  type: float
ignore_internal_state:

>   description: Specifies if state updates should ignore the internal state and always trigger the Home Assistant callback.

Reviewer: ```suggestion
  description: Specifies if telegrams should ignore the internal state and always trigger a Home Assistant state update.
```

  required: false
  type: string
reset_after:
  description: Reset back to OFF state after specified seconds.
  required: false

>   type: float

Reviewer: ```suggestion
  type: float
invert:
  description: Invert the telegrams payload before processing. This is applied before `context_timeout` or `reset_after` is evaluated.
  required: false
  type: boolean
  default: False
```

ignore_internal_state:
  description: Specifies if telegrams should ignore the internal state and always trigger a Home Assistant state update.

>   required: true

Reviewer: ```suggestion
  required: false
```
ignore_internal_state is optional

  type: boolean

>   default: False
> context_timeout:
>   description: The time in seconds between multiple identical telegram payloads would count towards the internal counter that is used for automations. Ex. You have automations in place that trigger your lights on button press and another set of lights if you click that button twice. This setting defines the time that a second button press would count toward, so if you set this 3.0 you can take up to 3 seconds in order to trigger the second button press. Maximum value is 10.0.

Reviewer: ```suggestion
  default: False
context_timeout:
  description: The time in seconds between multiple identical telegram payloads would count towards the internal counter that is used for automations. Ex. You have automations in place that trigger your lights on button press and another set of lights if you click that button twice. This setting defines the time that a second button press would count toward, so if you set this 3.0 you can take up to 3 seconds in order to trigger the second button press. If set `ignore_internal_state` will be set to `True` internally. Maximum value is 10.0.
```

  description: A type from the following table must be defined. The DPT of the group address should match the expected KNX DPT to be parsed correctly.
  required: true
  type: string
always_callback:

>   description: Defines if each update of the internal state should trigger a callback within Home Assistant no matter if the payload is equal.

Reviewer: ```suggestion
  description: Defines if telegrams with equal payload as the previously received telegram should trigger a state update within Home Assistant.
```

  description: A type from the following table must be defined. The DPT of the group address should match the expected KNX DPT to be parsed correctly.
  required: true
  type: string
always_callback:
  description: Defines if telegrams with equal payload as the previously received telegram should trigger a state update within Home Assistant.

>   type: boolean

Reviewer: ```suggestion
  required: false
  type: boolean
```
required key missing

  description: Separate KNX group address for retrieving the switch state. *DPT 1*
  required: false
  type: string

> reset_after:

Reviewer: ```suggestion
invert:
  description: Invert the telegrams payload before processing or sending. This is applied before `reset_after` is evaluated so the reset will send "1" to the bus.
  required: false
  type: boolean
  default: False
reset_after:
```
